src/assets/thirdPartyServices/snet/yolov3_object_detection/index.js

```diff
@@ -4,40 +4,50 @@ import Button from "@material-ui/core/Button";
 import Slider from "@material-ui/lab/Slider";
 
 import SNETImageUpload from "../../standardComponents/SNETImageUpload";
+import { Detector } from "./object_detection_pb_service";
+
+
+const initialUserInput = {
+  methodName: "Select a method",
+  img_path: undefined,
+  confidence: 0.5,
+  img_path: "",
+  methodName: "detect",
+};
 
 export default class YOLOv3ObjectDetection extends React.Component {
   constructor(props) {
     super(props);
     this.submitAction = this.submitAction.bind(this);
-    this.handleServiceName = this.handleServiceName.bind(this);
+
+    //this.handleServiceName = this.handleServiceName.bind(this);
     this.handleFormUpdate = this.handleFormUpdate.bind(this);
     this.getImageURL = this.getImageURL.bind(this);
-    this.getServiceMethods = this.getServiceMethods.bind(this);
+    //this.getServiceMethods = this.getServiceMethods.bind(this);
     this.handleSliderChange = this.handleSliderChange.bind(this);
 
     this.state = {
-      users_guide:
-        "https://github.com/singnet/dnn-model-services/blob/master/docs/users_guide/yolov3-object-detection.md",
+      ...initialUserInput,
+      users_guide: "https://github.com/singnet/dnn-model-services/blob/master/docs/users_guide/yolov3-object-detection.md",
       code_repo: "https://github.com/singnet/dnn-model-services/blob/master/Services/gRPC/yolov3-object-detection",
       reference: "https://pjreddie.com/darknet/yolo/",
-
       serviceName: "Detect",
-      methodName: "detect",
-
       model: "yolov3",
-      img_path: "",
-      confidence: 0.5,
-
       response: undefined,
     };
 
+    // TODO: Validate and Delete
+    /*
     this.isComplete = false;
```

Comment:
`isComplete` is being used in `render` method. We need to change that to it from the props.

Reply:
Made it available.